android/build.gradle

```diff
@@ -30,6 +30,6 @@ repositories {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:0.19.+'
+    compile 'com.facebook.react:react-native:0.32.+'
```

Comment:
In fact even after changing to this, there is something wrong with RN 0.33.

```xml
  <TabLayout>
    <Tab name="Tab 1" />
    <Tab name="Tab 2" />
  </TabLayout>
```

![image](https://cloud.githubusercontent.com/assets/1065299/18444550/83bcf02c-78ce-11e6-82fa-e9959799e96f.png)

I have confirmed that the test project indeed uses the RN 0.33 aar file. Do you need me to create a separate ticket for this?